README.md

```diff
@@ -1,7 +1,166 @@
 # Lively Docs
 
-### Initializing the Development Environment
+## Initializing the Development Environment
+
 1. `npm install`
 1. `bower install`
 1. `cp application/config.dist.js application/config.js`
 1. `gulp watch`
+
+## Configuration
+
+### Top Level Configuration
+
+The top level configuration for Lively is stored at `application/config.js`. An example is provided in `application/config.dist.js` and below:
+```
+{
+    lively : {
+        hostname : 'localhost',
+        port     : 9001
+    },
+    apis : {
+        github : require('./config/config.github')
+    }
+}
+```
+
+#### lively
+
+The `lively` configuration block defines the app's behavior.
+
+- **hostname**: A `string` containing the hostname where Lively will be installed
+- **port**: An `integer` containing the port for Lively to listen on
+
+#### apis
+
+The `apis` configuration block tells Lively where to find configuration files for the APIs you want to use with it. The recommended structure for these is to create folders in `application/config` for each API and use `require()` to include them (see example above).
+
+### API Configuration
+
+Each API has a configuration that tells Lively how to make requests and what methods are available. An example of implementing an API configuration for GitHub is provided at `application/config/config.github.js`.
+
+```
+{
+    'name'    : 'GitHub v3 API',
+    'logo'    : '/images/logos/github-mark.png',
+    'summary' : 'Summary text',
+    'api'     : {
+        'hostname' : 'api.github.com',
+        'port'     : 443,
+        'secure'   : true
+    },
+    'oauth2' : {
+        'type'         : 'authorization-code',
+        'hostname'     : 'github.com',
+        'port'         : 443,
+        'secure'       : true,
+        'authorizeUrl' : '/login/oauth/authorize',
+        'tokenUrl'     : '/login/oauth/access_token',
+        'tokenParam'   : 'token'
+    },
+    'resources' : [
+        require('./github/activity'),
+        require('./github/gists'),
+        require('./github/git_data'),
+        {...}
+    ]
+}
+```
+
+- **name**: A `string` containing the display name for the API.
+- **logo**: A `string` containing a valid path to an image to display for the logo
+- **summary**: A `string` containing the summary that will be displayed by default when you click on an API
+- **api**: An `Object` containing connection details for API calls (see below)
+- **oauth2**: An `Object` containing the connection and authentication  information for OAuth2 APIs (see below)
+- **resources**: An array of `Object` containing a unique key and an resource configuration object for each resource. Resources can be nested arbitrarily deep, see the resource configuration section for more details. The recommended structure is to create folders for each top level resource inside your API configuration folder.
```

Comment:
> resources: An array of Object containing a unique key and an resource configuration object for each resource. 

Knowing the structure of a resource object, I don't understand what is meant by "containing a unique key." Can you clarify?